frontend/packages/data-layout/src/layout.tsx

```diff
@@ -24,23 +24,23 @@ interface DataLayout {
   displayValue: () => any;
   /** Loading state of the layout. This is true when data is being hydrated. */
   isLoading: boolean;
-  /** Error state of the layout. This will be a message containing the error encountered when trying to hydrate the lyaout. */
+  /** Error state of the layout. This will be a message containing the error encountered when trying to hydrate the layout. */
   error: ClutchError;
 }
 
 /**
  * Use a registered data layout.
  * 
- * If a hydrate function has been specified this and the layout's data has not been set this will populate it's data
- * on the first invocation. If the layout has a cache key set to true and also has existing data hydrate will not be invoked.
+ * If a hydrate function has been specified this and the layout's data has not been set and shouldHydrate is true this will 
+ * populate it's data on the first invocation. If the layout has a cache key set to true and also has existing data OR
+ * shouldHydrate is false, hydrate will not be invoked.
 
  * @param key The name of the layout registered with the manager.
+ * @param shouldHydrate An option to not hydrate if we don't want to.
  */
-const useDataLayout = (key: string): DataLayout => {
+const useDataLayout = (key: string, shouldHydrate: boolean = true): DataLayout => {
```

Comment:
let's stick with an options object rather than adding an individual parameter in this case. it will read clearer on invocation and give us room to add more options later if needed.

`useDataLayout("foo", false);`
vs
`useDataLayout("foo", {hydrate: false}`

https://stackoverflow.com/questions/12826977/multiple-arguments-vs-options-object